### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by krishvoor**
This PR Aims at providing a fix for #6 by updating the relevant Documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Run this code from either Mac or Linux (Intel) system.
 - **Git**: Please refer to the following [link](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) for instructions
 on installing `git` for Linux and Mac.
 - **Terraform**: You'll need to use version `0.12.20`. Please refer to the following [link](https://learn.hashicorp.com/terraform/getting-started/install.html) for instructions on installing `terraform` for Linux and Mac.
+- **Terraform Providers**: Please ensure terraform providers are built and installed on KVM host. You can follow the [Build Terraform Providers](docs/terraform-provider-build.md) guide.
```
Comment (yussufsh) [inline README.md]: KVM host -> Terraform client machine

File: docs/terraform-provider-build.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Steps to build terraform providers
+
+Follow below steps to setup terraform providers. These steps are required to be followed before running the automation.
+
```
Comment (yussufsh) [inline docs/terraform-provider-build.md:4]: A note to follow below sections depending on the client arch would help.

File: README.md (hunk 1 above)
Comment (krishvoor) [reply README.md]: Updated

File: docs/terraform-provider-build.md (hunk 2 above)
Comment (krishvoor) [reply docs/terraform-provider-build.md:4]: Updated documentation to emphasize the steps required to be followed based on Terraform Client Machine Architecture.

**Review by yussufsh — APPROVED** [review]
LGTM

**Review by bpradipt — APPROVED** [review]
lgtm
Thanks @krishvoor

**Pull request merged**